olm-catalog/serverless-operator/project.yaml

```diff
@@ -37,9 +37,9 @@ dependencies:
   net_istio_artifacts_branch: release-1.7
   maistra: 2.1.0
 
-  eventing: knative-v1.6
+  eventing: knative-v1.7
```

Comment:
Our backports are now merged...

Reply:
I'm trying https://github.com/openshift-knative/serverless-operator/pull/1888 first. If this doesn't work out, I'll continue here.